packages/edit-site/src/components/keyboard-shortcut-help-modal/config.js

```diff
@@ -46,4 +46,8 @@ export const textFormattingShortcuts = [
 			'Convert the current paragraph or heading to a heading of level 1 to 6.'
 		),
 	},
+	{
+		keyCombination: { modifier: 'primaryShift', character: 'SPACE' },
```

Comment:
Can we use a literal space here as you did below for the rich text shortcut?

Reply:
Do you mean to use `\u00a0` instead of `SPACE`? In that case, the shortcut modal will display as below.

![image](https://github.com/WordPress/gutenberg/assets/54422211/9ecf7732-b04a-4866-8855-25af947eee43)

Reply:
Ah, this is for displaying the shortcut? Sorry it wasn't clear from the code.